templates/glance/config/00-config.conf

```diff
@@ -76,7 +82,7 @@ config_file = /etc/glance/glance-api-paste.ini
 flavor = keystone
 
 [os_glance_staging_store]
-filesystem_store_datadir = /var/lib/glance/os_glance_staging_store/
+filesystem_store_datadir = /var/lib/glance/image-cache
```

Comment:
This change will be undone in a future patch set.

Reply:
Done